frontend/public/components/_project-overview.scss

```diff
@@ -64,6 +64,7 @@
     }
 
     &::after {
+      align-self: center;
       content: '\f054';
```

Comment:
Since you've already got an unrelated change in here... This should make use of [the var](https://github.com/FortAwesome/Font-Awesome/blob/v4.7.0/scss/_variables.scss#L164) instead of directly using hex.

Reply:
Pulled into separate pr 
https://github.com/openshift/console/pull/608